data/mods/Aftershock/mobs/robots.json

```
@@ -279,23 +279,23 @@
     "id": "mon_hazmatbot",
     "type": "MONSTER",
     "copy-from": "mon_utilibot",
-    "name": "hazmat bot",
-    "description": "A utility robot designed for cleaning up waste material in hazardous conditions."
+    "name": "TsKBEM Likvidator",
+    "description": "The Likvidator was designed by the TsKBEM design bureau for the Soviet Union as an answer to liquidators post-Chernobyl.  Rather than using humans, easily suspectible to radiation sickness and hard to repair, the Likvidator would do the duties of taking care of toxic hazardous waste instead.  Likvidators are roughly human-sized, bipedal bulky machines; crafted for a utilitarian purpose of cleaning up toxic waste rather than aesthetic."
   },
   {
     "id": "mon_utilibot_butler",
     "type": "MONSTER",
     "copy-from": "mon_utilibot",
-    "name": "butler-bot",
+    "name": { "str": "ENIAC Famulus", "str_pl": "ENIAC Famuli" },
     "color": "dark_gray",
-    "description": "A luxury model utility robot for domestic use."
+    "description": "Top of the line and luxury built, the Famulus is a humaniform robot, derived from the humaniform schematics by Wraitheon with slight alterations to give it a more human appearance.  It is designed to be a robotic servant, its AI tailored for the needs for each customer; and arguably one of the few commercial robots to pass the turing test."
   },
   {
     "id": "mon_utilibot_const",
     "type": "MONSTER",
     "copy-from": "mon_utilibot",
-    "name": "construction robot",
-    "description": "One of the many models of construction robot formerly in use by government agencies and private corporations.  It is equipped with an integrated welder, flashlight, nailgun, and jackhammer.",
+    "name": "TsKBEM Udarnik",
+    "description": "One of the many challenges that the Soviet Union faced during its industrialization was that the model of shock brigades were unsustainable.  However, instead of slowing down, the answer that the Soviet Union used for this was the Udarnik.  Designed by the TsKBEM design bureau, it was made to replicate the zeal of strike workers, without the drawback of breaks, pay, or food.  This, along with in-built construction tools meant that the required work to compete with other world powers could be fulfilled.",
```

```suggestion
    "description": "One of the many challenges that the Soviet Union faced during its industrialization was the unsustainability of its traditional shock brigade model.  Instead of slowing down, the Soviet Union created the Udarnik.  Designed by the TsKBEM bureau, it was made to replicate the (supposed) zeal of Soviet workers, without the drawbacks of needing breaks, food, and medical care.  This, along with in-built construction tools, meant that the required work to compete with other world powers could be fulfilled.",
```
I'm not sure about "shock brigades" except in a military context, and "strike workers" does not communicate willingness to work. The last sentence could also be improved, but I'm not sure how.